telemetry_test.go

```diff
@@ -106,7 +106,10 @@ func TestDisableTelemetry(t *testing.T) {
 	if err = authenticateWithConfig(sc); err != nil {
 		t.Fatal(err)
 	}
-	if _, err = sc.Query("select 1", nil); err != nil {
-		t.Fatal(err)
+	if !sc.cfg.DisableTelemetry {
+		t.Errorf("DisableTelemetry should be true. DisableTelemetry: %v", sc.cfg.DisableTelemetry)
+	}
+	if sc.telemetry.enabled {
+		t.Errorf("telemetry should be disabled.")
```

Comment:
Can you also add a test that it is enabled when it is set to true? Also what's the default value?

Reply:
The test case for telemetry enabled has been added.
The default value of DisableTelemetry is false, i.e. telemetry is always enabled.